worker.js

```diff
@@ -0,0 +1,56 @@
+var Sway = require('sway')
```

Comment:
yeah, webpack is pretty awesome.. the only hiccup was that I needed to add 'json' as a type.
so that `require('something.json')` works as expected (from somewhere in `sway`)
I'm going to add an test.html file that tests some *very* basic functionality of sway, but via a Web Worker, then perhaps you'd consider a merge :wink: